Instagram/View Controllers/CommentsViewController.m

```diff
@@ -0,0 +1,59 @@
+//
+//  CommentsViewController.m
+//  Instagram
+//
+//  Created by Megan Miller on 6/28/22.
+//
+
+#import "CommentsViewController.h"
+#import "CommentCell.h"
+
+@interface CommentsViewController () <UITableViewDelegate, UITableViewDataSource>
+
+@property (weak, nonatomic) IBOutlet UITextField *commentField;
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (strong, nonatomic) NSMutableArray *comments;
+
+@end
+
+@implementation CommentsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+    
+    [self fetchComments];
+}
+
+- (void)fetchComments {
+    self.comments = self.post.comments;
+    [self.tableView reloadData];
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
```

Comment:
Protip: A useful way to organize your code is to use `#pragma mark` to create sections. Xcode gives you a nice UI for this too.

We generally will separate methods specific to a protocol (i.e. `UITableViewDataSource`) into a section using pragma mark

```
#pragma mark UITableViewDataSource
- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section { 
    ...
}


- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
   ....
}

etc
```
```